Networking/Sources/Networking/Networking.swift

```diff
@@ -1,6 +1,49 @@
+import Alamofire
+import Foundation
+
 public struct Networking {
     public private(set) var text = "Hello, World!"
 
     public init() {
     }
+    
+    public func perform<M: Decodable, T: TargetType>(target: T, responseClass: M.Type, completion: @escaping (Result<M?, NSError>) -> ()){
```

Comment:
```suggestion
    public func perform<M: Decodable, T: TargetType>(target: T, responseClass: M.Type, completion: @escaping (Result<M, Error>) -> ()){
```
- It's better to use `Error` instead of `NSError`
- Why do you need an optional value? I think it's not needed.